Update v0.4.17.rc0 release notes (#5058)
Napari/napari: docs/release/release_0_4_17.md

## Highlights

> ### Minimum blending for a better light theme experience

Author: I'm not sure if there's a desired style for highlights, but I kind of like sub-sections. Can change to a list or something else if desired.

## Highlights

This release is focused on documentation improvements and bug fixes, with few changes to napari’s API.
We picked out a few highlights, but keep reading below for the full list of changes!

> ### Minimum blending for a better light theme experience

Reviewer: ```suggestion
### Minimum blending and inverted colormaps
```

> We added a new minimum blending option in #4875 that is particularly useful for those

Author: @psobolewskiPhD : I think you're busy at the moment, so I added a short blurb about this new feature. But feel to suggest corrections and additions!

Reviewer: Thanks for the heads up and the opportunity ♥️ 
I expanded a bit to include mention of the inverted colormaps from @cleterrier 
I used suggestions, a bit awkwardly. Feel free and ignore of course or tweak as you see fit.

Author: Suggestions were perfect. I committed them then followed up with a little rewording.

> We added a new minimum blending option in #4875 that is particularly useful for those

Reviewer: ```suggestion
We added a new blending mode,  `minimum`,  in #4875,  along with some inverted colormaps by @cleterrier. The new mode and colormaps enable visualizing data as color intensity on white background. This is particularly nice when using the Light theme. For more inspiration see [#invertedLUT on Twitter](https://twitter.com/hashtag/invertedLUT)
```

> using the napari light theme.

Reviewer: ```suggestion
```

See [examples/minimum_blending.py](https://github.com/napari/napari/blob/main/examples/minimum_blending.py) for an example.

### Option to edit second shortcut in settings

> TODO

Author: @Czaki : could you suggest one or two sentences to highlight this work?

Reviewer 2: Improvement of customization napari by allow to edit second shortcuts for napari actions. 
This allow to have both numeric (1, 2 ,3 - related to button positions) and semantic (F - for fill) shortcuts defined from user interface.

Author: Thanks! I added this with a little rewording!

### `napari.imshow` as a convenient one-liner

> TODO

Author: @kcpevey : could you suggest one or two sentences (and maybe a short code example) to highlight this work?

Reviewer 3: A convenience method to load image data and return both a Viewer and the Layer (also accepts an existing viewer).
```
data = np.random.random((10, 15))
viewer, layer = napari.imshow(data)
```

Author: Thanks will add this soon.

### Versioned docs

> TODO

Author: @melissawm : I thought it would be great to highlight this work. If you have time to suggest one or two sentences, that would be ideal.

### Experimental support for Qt6

> We added some basic support for PyQt6 and PySide6 in #3707, though we expect some issues.

Author: @tlambert03 : I think you led the effort here. I attempted a short blurb, but feel free to suggest an alternative.

## Highlights

This release is focused on documentation improvements and bug fixes, with few changes to napari’s API.
We picked out a few highlights, but keep reading below for the full list of changes!

### Minimum blending and inverted colormaps

We added a new blending mode, `minimum`, in #4875, along with some inverted colormaps by @cleterrier.
The new mode and colormaps enable visualizing data as color intensities on a white background.
This is particularly nice when using the Light theme.
For more inspiration see [#invertedLUT on Twitter](https://twitter.com/hashtag/invertedLUT)
and the built-in napari example [examples/minimum_blending.py](https://github.com/napari/napari/blob/main/examples/minimum_blending.py).

### Option to edit second shortcut in preferences

We improved customization of napari preferences by allowing you to define a second,
alternative shortcut for each napari action.

> This allows you have both numeric (e.g. 1, 2, 3 - related to button positions) and semantic (F - for fill) shortcuts

Reviewer 4: ```suggestion
This allows you, for example, to have both numeric (e.g. 1, 2, 3 - related to button positions) and semantic (F - for fill) shortcuts
```

- add ci for asv benchmark suite (#4554)
- Disconnect a removed slider's _pull_label callback from the axis label change event  (#4557)
- Fixes/modifications to nested list model and Group in prep for layergroups (#4560)

> - [Automatic] Update albertosottile/darkdetect vendored module (#4561)

Author: I had to manually add this line. Based on the behavior described in #4562 I believe this was not included in v0.4.16, but I have not followed the tags/commits to understand why this was skipped by the release notes generation script. Maybe because [it tries to skips contribution from bots](https://github.com/napari/napari/blob/fe4dfb7b798a75f36df7182f66f74141744eb863/docs/release/generate_release_notes.py#L119)?

- Fix broken links in Napari Code of Conduct (#5005)
- Fix sys.path issue with subprocess relaunch in macOS (#5007)
- MAINT: update issue failing template (#5012)
- Add link to docs artifact to pull requests (#5014)
- [pre-commit.ci] pre-commit autoupdate (#5015)
- Feature: register a keyboard shortcut for `preserve_labels` checkbox (#5017)
- Add option to edit second shortcut in settings (#5018)
- Fix shift selection for points (#5022)
- Revert "Fix sys.path issue with subprocess relaunch in macOS" (#5027)
- Use Source object to track if a layer is duplicated (#5028)
- Revert "Revert "Fix sys.path issue with subprocess relaunch in macOS"" (#5029)
- Add release notes for 0.4.17 (#5031)
- MAINT: fix a couple of trans._. (#5034)
- Ensure non-negative edge width (#5035)

> - add #4954 to release notes (#5038)

Author: Not sure if we should include release note updates in the release notes, but I'm lazy and just kept them for now.
